app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] Foreign key already declared in the _User_ model, and hence no need to declare it again in a referenced model. Simply do something like:
```ruby
# app/models/like.rb
belongs_to :author, class_name: 'User'